app/controllers/v0/letters_generator_controller.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'lighthouse/letters_generator/service'
+require 'lighthouse/letters_generator/service_error'
+
+module V0
+  class LettersGeneratorController < ApplicationController
+    Raven.tags_context(team: 'benefits-claim-appeal-status', feature: 'letters-generator')
+    DOWNLOAD_PARAMS = %i[
+      id
+      format
+      militaryService
+      serviceConnectedDisabilities
+      serviceConnectedEvaluation
+      nonServiceConnectedPension
+      monthlyAward
+      unemployable
+      specialMonthlyCompensation
+      adaptedHousing
+      chapter35Eligibility
+      deathResultOfDisability
+      survivorsAward
+    ].freeze
+
+    def index
+      response = service.get_eligible_letter_types(@current_user.icn)
+      render json: response
+    rescue Lighthouse::LettersGenerator::ServiceError => e
+      render json: { title: e.title, message: e.message, status: e.status }.to_json
```

Comment:
If we rescue and render custom JSON here, it's no longer an error, and so it's possible it will skip handling by the `ExceptionHandling` concern (app/controllers/concerns/exception_handling.rb), included in the base `ApplicationController`. I'm not 100% certain of that though. What do you think?